python_scripts/01_tabular_data_exploration.py

```diff
@@ -41,8 +41,9 @@
 
 # %% [markdown]
 # We can look at the OpenML webpage to learn more about this dataset: http://www.openml.org/d/1590
-#
-# It tells us that the typical prediction task is to determine whether a person makes over 50K a year.
+
+# The goal with this data is to regress wages (i.e. whether a person makes over 50K a year)
```

Comment:
Do we want to employ the term "regress" when the problem will be a classification problem?

Reply:
Also we should avoid too much jargon for these kind of beginner-oriented notebooks (I know it is hard).

Having said that, I have to say I don't really know how common "regress" in non-technical English.